README.md

```diff
@@ -1 +1,74 @@
-# space-travelers-hub
\ No newline at end of file
+# React Project: Space Traveler's Hub
+
+This is a Space Traveler's Hub website. The website has 3 pages that lists rockets, missions, and the reserved rockets and booked missions.
+
+# Screen Shots
+![rockets](https://user-images.githubusercontent.com/49184579/169244577-7b5dcf7b-0609-43d5-acda-2a3905d8aaac.png)
+
+![missions](https://user-images.githubusercontent.com/49184579/169244592-69622ecd-1164-4416-a619-f817ca6fe29e.png)
+
+![my-profile](https://user-images.githubusercontent.com/49184579/169244603-bcad628c-a8d1-4cad-991e-6248c079a80f.png)
+
+
+
+## Built With &hearts;
+
+- ReactJs
+- Redux
+- Redux-thunk
+- Bootstrap
+
+## Live Demo
+
+[Github-page live](https://aboongm.github.io/space-travelers-hub/)
+
+[Netlify live](https://aboongm-space-travelers-hub.netlify.app/)
+
+### Prerequisites
+
+- Good knowledge on javascript objects
+- VSCode or other text editor installed. [Link to download VSCode](https://code.visualstudio.com/download)
+- Node package. [Link to download node](https://nodejs.org/en/download/)
+
+## Getting started
+
+to run the project locally, clone the project using the command
+
+`git clone git@github.com:aboongm/react-bookstore.git`
+
+### Run test
+
+navigate to the project directory, from GitHub,
+to test or run the application double click on the `index.html` file to open in your browser
+
+## Authors
+
+👤 **Ranjit Luwang**
+
+- GitHub: [@aboongm](https://github.com/aboongm)
+- Twitter: [@John_luang1](https://twitter.com/John_luang1)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/mayengbam-ranjit-luwang-31962418/)
+
+👤 **Said Rasinlic**
+
+- GitHub: [@GitHub/SaidRasinlic](https://github.com/SaidRasinlic)
+- Twitter: [@Twitter/SaidRasinlic](https://twitter.com/SaidRasinlic)
+- LinkedIn: [LinkedIn/SaidRasinlic](https://www.linkedin.com/in/saidrasinlic)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Microverse program
+- My standup team
+- Microverse team
+- My family's support
```

Comment:
- [ ] **Optional:** You could consider adding the **MIT** license as shown in the Microverse readme template.